synapseclient/models/mixins/table_components.py

```diff
@@ -64,23 +122,6 @@ class TableBase:
     _last_persistent_instance: None = None
     _columns_to_delete: Optional[Dict] = None
 
-    def _set_last_persistent_instance(self) -> None:
-        """Used to satisfy the usage in this mixin from the parent class."""
-
-    def to_synapse_request(self) -> Dict:
-        """Used to satisfy the usage in this mixin from the parent class."""
-
-    def fill_from_dict(self, entity: Dict, set_annotations: bool) -> None:
-        """Used to satisfy the usage in this mixin from the parent class."""
-
-    @property
-    def has_changed(self) -> bool:
-        """Used to satisfy the usage in this mixin from the parent class."""
-
-    @property
-    def has_columns_changed(self) -> bool:
-        """Used to satisfy the usage in this mixin from the parent class."""
-
```

Comment:
I was having issues where keeping these in was causing `self` to evaluate to `None` at runtime, and in the debugger it was hitting these types of functions (In the few spots I deleted these).

We'll need to figure out if we need to keep these or not

Reply:
I liked having these in as a reminder to implement them in a new model, but I suppose we don't technically _need_ them.